source/retryable-writes/retryable-writes.rst

```diff
@@ -533,7 +533,7 @@ specification does not define a format for such log messages.
 Command Monitoring
 ==================
 
-In accordance with the `Command Monitoring`_ specification, drivers MUST
```

Comment:
similar comment to above, re making the text on correlating events and new events for retries apply to log messages as well.